echo/wavefront-tracing-logger.go

```diff
@@ -0,0 +1,168 @@
+package echo
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/opentracing/opentracing-go"
+)
+
+//WfLogger custom implementation of Go standard Logger
+//Exposes the same functions as Standard Logger
+type WfLogger struct {
```

Comment:
Rename to `SpanLogger` to make it explicitly clear what sort of logging this enables.